modBoot/src/org/aion/Aion.java

```diff
@@ -294,6 +303,16 @@ private ShutdownThreadHolder(Thread zmqThread, IMineRunner nm, ProtocolProcessor
 
     }
 
+    public static String appendLogo(String value, String input) {
+        int leftPad = Math.round((44 - input.length()) / 2.0f) + 1;
+        StringBuilder padInput = new StringBuilder();
```

Comment:
I feel like it's a bit unproductive to use a `StringBuilder` only then to resort to string concatenation after adding the padding. You can initialize the builder with `value` instead and then append your spaces.
Also maybe worth considering apache commons for the padding: `StringUtils.repeat(' ', length)` instead of the for-loop

Reply:
i think this was the implementation taken from the rpc-serverswap branch, i simply refactored it... but i can still modify it here

Reply:
ok